### PR TITLE
Scavengers: fix display bugs

### PR DESCRIPTION
**Pull request by sparkychildcharlie**
I guess this has been in since the multiple hosts update... which was a long time ago... oops i guess?

Nyaaa~ :3

### CODE REVIEW

File: server/chat-plugins/scavengers.ts [hunk 1]
```diff
@@ -1424,7 +1424,7 @@ const ScavengerCommands: ChatCommands = {
 			room,
 			gameType === 'official' || gameType === 'recycled'
 		);
-		if (!hosts) return this.errorReply("The user(s) you specified as the host is not online, or is not in the room.");
+		if (!hosts.length) return this.errorReply("The user(s) you specified as the host is not online, or is not in the room.");
```
Comment (AnnikaCodes) [inline server/chat-plugins/scavengers.ts:1427]:
```suggestion
		if (!hosts.length) {
				return this.errorReply("One or more of the user(s) you specified as the host is offline or not in the room.");
		}
```

File: server/chat-plugins/scavengers.ts [hunk 2]
```diff
@@ -1733,7 +1733,7 @@ const ScavengerCommands: ChatCommands = {
 		} else {
 			const [hostsArray, ...params] = target.split('|');
 			const hosts = ScavengerHunt.parseHosts(hostsArray.split(/[,;]/), room);
-			if (!hosts) return this.errorReply("The user(s) you specified as the host is not online, or is not in the room.");
+			if (!hosts.length) return this.errorReply("The user(s) you specified as the host is not online, or is not in the room.");
```
Comment (AnnikaCodes) [inline server/chat-plugins/scavengers.ts:1736]:
```suggestion
			if (!hosts.length) {
					return this.errorReply("One or more of the user(s) you specified as the host(s) is offline or not in the room.");
			}
```

**Comment by Zarel** [comment]
@sparkychildcharlie you've marked those as resolved but it's a day later and your fix isn't in yet...

**Comment by Zarel** [comment]
I'm drafting until it's fixed.

**Comment by sparkychildcharlie** [comment]
Took me like 4 days to realize there was a ready for review button

**Comment by Zarel** [comment]
Ah, it's fine to take a while. I was just afraid you marking things as resolved was a misclick, since Mia often does it and then forgets to actually fix the thing.

**Pull request merged**